quaddtype/pyproject.toml

```diff
@@ -37,6 +37,12 @@ test = [
     "pytest-run-parallel"
 ]
 
+docs = [
+    "sphinx",
```

Comment:
@ngoldbaum (didn't explored in deep yet) Would this startup setup be extendable to your work (https://unyt.readthedocs.io/en/stable/)